### ISSUE TITLE
Problem mask "+7(###)#######"

### ISSUE THREAD

**Issue by yura-f** [VicMikhailau/MaskedEditText#8]
Version 2.0
If you create a mask like this:
```
MaskedFormatter formatter = new MaskedFormatter("+7(###)#######");
editText.addTextChangedListener(new MaskedWatcher(formatter, editText));
```

The number 7 is not entered!

**Comment by Mixser** [comment]
@Z-13 Ok, I understood and fix it. But I have a question. Fox example if we use a next mask: "+7(###)#######". If user type '+' the output will be '+', '+7' => '+7'. If the user type '1' to empty input what is right output by your opinion: "+7(1" or "+" ?

**Comment by yura-f** [comment]
@Mixser I think that mask is like this: "+7(###)#######" and user entered 1 then show "+7(1". Because, the mask has already "+7". :)

**Comment by Mixser** [comment]
#10 fix this problem

**Comment by yura-f** [comment]
@Mixser thx! :+1:

**Comment by VicMikhailau** [comment]
Fixed in 2.0.1 library version.
